### ISSUE TITLE
Implementation of jpsTools

### ISSUE THREAD

**Issue by gjaeger** [JuPedSim/jpseditor#177]
Does it make sense to implement [jpsTools](https://github.com/bsmoehring/jpsTools) in JPSeditor? The repository contains a tool for converting osm-data into jupedsim geometry files.

**Comment by chraibi** [comment]
Yes, it makes sense to implement the *functionality* of jpsTools in JPSeditor.
